workspaces/orchestrator/plugins/orchestrator-backend/src/service/api/v2.ts

```diff
@@ -163,12 +163,11 @@ export class V2 {
     workflowId: string,
     businessKey: string | undefined,
   ): Promise<ExecuteWorkflowResponseDTO> {
-    if (Object.keys(executeWorkflowRequestDTO?.inputData).length === 0) {
+    if (!executeWorkflowRequestDTO?.inputData) {
```

Comment:
inputData is a required field in the spec. You need to fix the spec too

Reply:
if it's required why are we checking this?

Reply:
Revisisted it
I think the inputData should be optional, so I changed the spec for it to be optional
Actually useful, a workflow without parameters can be run without need to pass a body:
curl -X POST http://localhost:7007/api/orchestrator/v2/workflows/hello_world/execute